dotCMS/src/main/java/com/dotcms/contenttype/business/uniquefields/extratable/UniqueFieldDataBaseUtil.java

```diff
@@ -144,13 +145,18 @@ public void insertWithHash(final String key, final Map<String, Object> supportin
         new DotConnect().setSQL(INSERT_SQL_WIT_HASH).addParam(key).addJSONParam(supportingValues).loadObjectResults();
     }
 
-    @WrapInTransaction
```

Comment:
Why?

Reply:
@freddyDOTCMS what happens if, for instance, you have 3 unique fields?

Let's say the first one is saved correctly, the second one is saved correctly too, but the third one fails. Does that mean the first two changes will be kept? Or is another method rolling those changes back?

Reply:
if you want to do this in a separated connection you have to do:
```
@WrapInTransaction(externalize=true)
```

This will take a new connection and do the transaction for each call